shared/test/test_table_limits.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'test_helper'
+require 'fake_redis'
```

Comment:
The gem documentation says to 'fakeredis' not `fake_redis`: https://github.com/guilleiguaran/fakeredis/blob/master/README.md#usage Since you say this test is hitting local fake redis, I would assume that this is only working because a different test case requires 'fakeredis'.